numpy/core/src/multiarray/abstractdtypes.h

```diff
@@ -28,11 +28,19 @@ initialize_and_map_pytypes_to_dtypes(void);
  * @param arr The array into which the Python object was converted.
  * @param[in,out] **dtype A pointer to the array's DType, if not NULL it will be
  *        replaced with the abstract DType.
- * @return 0 if the `obj` was not a python scalar, and 1 if it was.
+ * @param replace_int_arr When True (only in ufunc code!), replaces a
+ *        non-standard integer array with a normalized one.
```

Comment:
I cannot follow this explanation! What warnings are broken?

Reply:
The "NEP 50 changes things warnings" won't always work, but since it probably never matter in practice, I guess who cares...